### PR TITLE
Add fts-rest and pygsi

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes, this is ready for review. Windows fails due to a `noarch` package having poor support for Windows and can be ignored.

File: recipes/fts-rest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  host:
+    - python <3
+    - pip
+  run:
+    - python <3
+    - pycurl
+    - m2crypto >=0.16
+    - requests
+    - sqlalchemy
+    - setuptools_scm
```
Comment (xhochy) [inline recipes/fts-rest/meta.yaml]: Shouldn't this also be a `host` requirement?

Comment (chrisburr) [reply recipes/fts-rest/meta.yaml]: So far as I can tell it's only needed at runtime unless there is any "magic" that comes from having it in `host` that I'm unaware of?

Comment (xhochy) [reply recipes/fts-rest/meta.yaml]: I've looked at https://github.com/cern-fts/fts-rest/blob/master/setup.py and cannot see that it requires `setuptools_scm` at all. Maybe this is a transitive dependency?

Comment (chrisburr) [reply recipes/fts-rest/meta.yaml]: It's needed for `pkg_resources` ([used here](https://github.com/cern-fts/fts-rest/blob/1dee4fa72d4b04d0a2c3cc0f56ff369d850dfd39/src/fts3/__init__.py)).

Comment (xhochy) [reply recipes/fts-rest/meta.yaml]: `pkg_resources` is part of `setuptools`, not `setuptools_scm` ;)

**Pull request merged**